### DISCUSSION TITLE
Best way to keep hkknx running in the background on OS X?

### DISCUSSION THREAD

**Discussion by dotgrid** [brutella/hkknx-public#146] [General]
Hi all. What's the most effective way to keep the app running at all times on OS X? I have it open in the Terminal and it's working great.

**Comment by brutella** [comment]
`launchd` might be a good option. See [Creating Launch Daemons and Agents](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html)

**Reply by dotgrid** [reply]
Thanks! I'll give that a try and I'll share the .plist file here if I get it working :) 

@brutella Have you considered submitting hkknx to Homebrew? Would make install, updates (and running as a service) nice and simple on Mac OS.
